ports/stm32f3/boards.c

```diff
@@ -30,7 +30,7 @@
 // MACRO TYPEDEF CONSTANT ENUM DECLARATION
 //--------------------------------------------------------------------+
 
-#define STM32_UUID ((uint32_t *)0x1FFFF7AC)
+#define STM32_UUID ((uint32_t *)0x1FFFF7ACUL)
```

Comment:
can you try to add volatile as follow and keep memcpy to see if that make gcc11 happy 

```C
#define STM32_UUID ((volatile uint32_t *)0x1FFFF7AC)
```

Reply:
I tried this already and this don't work.